PhysicsTools/PatUtils/plugins/L1ECALPrefiringWeightProducer.cc

```diff
@@ -214,12 +214,12 @@ L1ECALPrefiringWeightProducer::fillDescriptions(edm::ConfigurationDescriptions&
   desc.add<edm::InputTag>("TheJets", edm::InputTag("slimmedJets"));
   desc.add<std::string>("L1Maps","L1PrefiringMaps.root");
   desc.add<std::string>("DataEra", "2017BtoF");
-  desc.add<bool>("UseJetEMPt", true);
+  desc.add<bool>("UseJetEMPt", false);
   desc.add<double>("PrefiringRateSystematicUncty",0.2);
-  desc.add<bool>("SkipWarnings", true);
-  
-  descriptions.add("l1ECALPrefiringWeightProducer",desc);
+  desc.add<bool>("SkipWarnings", false);
```

Comment:
Why did you change the default value into this more verbose one? I think it can be safer to have the "SkipWarnings" true by default, and only enable warnings if needed